### PR TITLE
[fix][doc] Tips markup

### PR DESCRIPTION
**Pull request by tisonkun**
This is a follow-up to https://github.com/apache/pulsar/pull/18101.

### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete` 

### Matching PR in forked repository

PR in forked repository:

### CODE REVIEW

File: site2/docs/reference-cli-tools.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ Pulsar offers several command-line tools that you can use for managing Pulsar in
 * `pulsar-shell`
 * `bookkeeper`
 
-::: tip
+:::tip
 
-For the latest and complete information about command-line tools, including commands, flags, descriptions, and more information, see [Pulsar Reference](https://pulsar.apache.org/reference).
+For the latest and complete information about command-line tools, including commands, flags, descriptions, and more information, see [Pulsar Reference](/reference).
```
Comment (momo-jun) [inline site2/docs/reference-cli-tools.md:19]: IIRC, `/` should be enforced at the end, like `/reference/`.
BTW, there is another occurrence of `https://pulsar.apache.org/reference/` in the `reference-configuration.md` file.

Comment (tisonkun) [reply site2/docs/reference-cli-tools.md:19]: Let me revert this change here. We can do it in a separate PR.

Comment (tisonkun) [inline site2/docs/reference-cli-tools.md:19]:
```suggestion
For the latest and complete information about command-line tools, including commands, flags, descriptions, and more information, see [Pulsar Reference](https://pulsar.apache.org/reference).
```

Comment (tisonkun) [reply site2/docs/reference-cli-tools.md]: Reverted.

**Pull request merged**